modules/svdb/merge/main.nf

```diff
@@ -22,14 +22,20 @@ process SVDB_MERGE {
     def args   = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
     def input  = ""
-    for (int index = 0; index < vcfs.size(); index++) {
-        input += " ${vcfs[index]}:${priority[index]}"
+    if(priority) {
+        prio = "--priority ${priority.join(',')}"
+        for (int index = 0; index < vcfs.size(); index++) {
+            input += " ${vcfs[index]}:${priority[index]}"
+        }
+    } else {
+        prio = ""
+        input = "${vcfs.join(" ")}"
```

Comment:
You could make this the default option and then just overwrite these variabels if `priority` is set. That is if you want to save a few lines of code. Doesn't really matter 😄

Reply:
Good point! Changed it now 😄